bin/coffeelint.rb

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env ruby
+#
+# This file was generated by Bundler.
+#
+# The application 'coffeelint.rb' is installed as part of a gem, and
+# this file is here to facilitate running it.
+#
+
+require 'pathname'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.